auditbeat/module/file_integrity/mime.go

```diff
@@ -0,0 +1,53 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Elasticsearch B.V. licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package file_integrity
+
+import (
+	"github.com/h2non/filetype"
+
+	"github.com/elastic/beats/v7/libbeat/common/file"
+)
+
+const (
+	// Size for mime detection, office file
+	// detection requires ~8kb to detect properly
+	headerSize = 8192
+)
+
+// this does a best-effort to get the file type, if no
```

Comment:
I like to follow the godoc conventions even for unexported functions like `// getMimeType ...`.